quantum/action_util.h

```diff
@@ -18,22 +18,45 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include <stdint.h>
+#include <stddef.h>
 #include "report.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-extern report_keyboard_t *keyboard_report;
+#if !defined(QMK_KEYS_PER_SCAN) && defined(REGISTER_MULTIPLE_KEYEVENTS_ENABLE)
+#    pragma message "REGISTER_MULTIPLE_KEYEVENTS_ENABLE needs QMK_KEYS_PER_SCAN defined to have any effect, setting default of 4."
+#    define QMK_KEYS_PER_SCAN 4
+#endif
+
+#if !defined(QMK_KEYS_PER_SCAN)
+#    define UNREGISTER_KEYCODES_BUFFER_SIZE 1
```

Comment:
not documented?

Reply:
Not configurable?

Reply:
Actually I'll would like to drop `QMK_KEYS_PER_SCAN` entirely and give the main keyboard loop a facelift :-), will add documentation nevertheless.